dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -53,11 +48,11 @@ then
     
     DBT_PROJECT_PATH='reporting'
     # This is a Slim CI syntax used to "run" only modified and downstream models
-    DBT_RUN_OPTIONS="-m state:modified+ @state:modified,1+test_type:data --defer --state $WORKSPACE/manifest" 
```

Comment:
excluded 1+test_type:data from models creation tag as it was suggested to use in v0.18.1 as an alternative to not being able to use --defer state in tests, we are now on v0.19.1 so removing it.